ext/standard/tests/hrtime/hrtime.phpt

```diff
@@ -0,0 +1,20 @@
+--TEST--
+Test hrtime() returns nanoseconds
+--FILE--
+<?php
+
+$time = hrtime();
+usleep(500000);
```

Comment:
Not systems, but process on some system. Sleep functions don't guarantee the exact time you pass, and the process needs to wake up, etc. The resolution of the sleep function is not guaranteed, a busy wait is more reliable in this purpose.